src/OpenTelemetry/Trace/Export/SpanExporter.cs

```diff
@@ -16,59 +16,46 @@
 
 namespace OpenTelemetry.Trace.Export
 {
-    using System;
+    using System.Collections.Generic;
     using System.Threading;
     using System.Threading.Tasks;
+    using OpenTelemetry.Trace;
 
-    public sealed class SpanExporter : SpanExporterBase
+    /// <summary>
+    /// SpanExporter base class.
+    /// </summary>
+    public abstract class SpanExporter
     {
-        private readonly Thread workerThread;
-
-        private readonly SpanExporterWorker worker;
-
-        internal SpanExporter(SpanExporterWorker worker)
-        {
-            this.worker = worker;
-            this.workerThread = new Thread(worker.Run)
-            {
-                IsBackground = true,
-                Name = "SpanExporter",
-            };
-            this.workerThread.Start();
-        }
-
-        internal Thread ServiceExporterThread => this.workerThread;
-
-        public static ISpanExporter Create(int bufferSize = 32, TimeSpan? scheduleDelay = null)
-        {
-            var worker = new SpanExporterWorker(bufferSize, scheduleDelay ?? TimeSpan.FromSeconds(5));
-            return new SpanExporter(worker);
-        }
-
-        public override void AddSpan(Span span)
-        {
-            this.worker.AddSpan(span);
-        }
-
-        /// <inheritdoc/>
-        public override Task ExportAsync(Span export, CancellationToken token)
-        {
-            return this.worker.ExportAsync(export, token);
-        }
-
-        public override void RegisterHandler(string name, IHandler handler)
+        public enum ExportResult
         {
-            this.worker.RegisterHandler(name, handler);
+            /// <summary>
+            /// Batch is successfully exported.
```

Comment:
Why batch is in the name of an enum in the abstract exporter?

Reply:
exporters always export list of spans. Batch vs non-batch exists only in processor.